packages/app/src/components/inspector/PatternInspector.tsx

```diff
@@ -0,0 +1,242 @@
+import Sketch from '@sketch-hq/sketch-file-format-ts';
+import {
+  Select,
+  Spacer,
+  Slider,
+  InputField,
+  getPatternBackground,
+  SketchPattern,
+  getPatternSize,
+  SUPPORTED_FILE_TYPES,
+} from 'noya-designsystem';
+import { memo, useCallback, useMemo, useState, DragEvent } from 'react';
+import styled from 'styled-components';
+import * as InspectorPrimitives from './InspectorPrimitives';
+import { fileOpen } from 'browser-fs-access';
+import { uuid } from 'noya-renderer';
+import { FileMap } from 'noya-sketch-file';
+import { useHover } from 'noya-designsystem/src/hooks/useHover';
+import { useFileDropTarget } from '../../hooks/useFileDropTarget';
+
+const Column = styled.div(({ theme }) => ({
+  flex: '1',
+  display: 'flex',
+  flexDirection: 'column',
+}));
+
+const Container = styled.div<{
+  background: string;
+  backgroundSize: string;
+  repeat: boolean;
+  isActive: boolean;
+}>(({ theme, background, backgroundSize, repeat, isActive }) => {
+  const dragOverlayColor =
```

Comment:
This is clever, but will break if we change the color to a hex code or something. Let's add a new theme color for this overlay color